plugin.video.themoviedb.helper/resources/lib/container.py

```diff
@@ -0,0 +1,261 @@
+import sys
+import utils
+import xbmc
+import xbmcgui
+import xbmcaddon
+import xbmcplugin
+from tmdb import TMDb
+from omdb import OMDb
+from kodilibrary import KodiLibrary
+from listitem import ListItem
+from globals import LANGUAGES, BASEDIR, TYPE_CONVERSION, TMDB_LISTS, TMDB_CATEGORIES, APPEND_TO_RESPONSE
+from urlparse import parse_qsl
```

Comment:
py3 compliance pls change all instances of `from urlparse import parse_qsl` to 'from urllib.parse import parse_qsl`